ska_trend/centroid_dashboard/index_template.html

```diff
@@ -0,0 +1,208 @@
+<html>
+
+<head>
+    <title>Obsid {{obs.obsid}}</title>
+
+    <style type="text/css">
+        body {
+            margin-left: 20px;
+            background-color: lightblue;
+        }
+
+        * {
+            box-sizing: border-box;
+        }
+
+        div.border {
+            background-color: white;
+            padding-left: 5px;
+            padding-right: 5px;
+            padding-bottom: 5px;
+            border-style: solid;
+            border-width: 2px;
+            border-radius: 10px;
+        }
+
+        /* Create three equal columns that floats next to each other */
+        .column1 {
+            float: left;
+            width: 300px;
+            padding: 5px;
+        }
+
+        .column2 {
+            float: left;
+            width: 640px;
+            padding: 5px;
+        }
+
+        .column3 {
+            float: left;
+            width: 500px;
+            padding: 5px;
+        }
+
+        /* Clear floats after the columns */
+        .row:after {
+            content: "";
+            display: table;
+            clear: both;
+        }
+
+        table.single-line {
+            border-width: medium;
+            border-spacing: 2px;
+            border-style: none;
+            border-color: gray;
+            border-collapse: collapse;
+            background-color: white;
+        }
+
+        table.single-line th {
+            border-width: 2px;
+            padding: 2px 3px 2px 3px;
+            border-style: inset;
+            border-color: gray;
+            background-color: white;
+        }
+
+        table.single-line td {
+            border-width: 2px;
+            padding: 2px 3px 2px 3px;
+            border-style: inset;
+            border-color: gray;
+            background-color: white;
+        }
+    </style>
+</head>
+
+<body>
+
+    <div class="column1">
+
+        <div class="border" style="padding: 10px; margin-bottom: .5cm">
+            <span id="label" style="font-size:150%; font-weight:bold;">
+                {% if obs.obs_prev %}
+                <a href="../../../{{obs.obs_prev.report_subdir}}/index.html">
+                    Prev</a>
+                {% else %}
+                None
+                {% endif %}
+                - {{obs.obsid}} -
+                {% if obs.obs_next %}
+                <a href="../../../{{obs.obs_next.report_subdir}}/index.html">
+                    Next</a>
+                {% else %}
+                None
+                {% endif %}
+            </span>
+        </div>
+
+        <div class="border" style="float: left; margin-bottom: .5cm;">
+
+            <div>
+                <h2>Observation Details</h2>
+
+                <pre>
+ObsID <a href="{{MICA_PORTAL}}/?obsid_or_date={{obs.obsid}}" target="_blank">{{obs.obsid}}</a>
```

Comment:
For these obsid_or_date click throughs to mica, it isn't really a centroid dashboard thing, but do you want to just go ahead and either set these to date or include a load name in the URL and we'll figure out the rest on the mica and kadi-apps side?

Reply:
Good idea. We just need a convention. Would <load_name>/<obsid> be OK? I.e. wherever you might put an obsid, you could instead using something like `MAR3125A/42522`?

Reply:
The "loadname/obsid" string could work, though for a CGI I'm inclined to just be explicit in the labels for what the supplied things are and go with "obsid=42522&load_name=MAR3125A".